Coremelysis/Coremelysis/MainView/MainViewModel.swift

```diff
@@ -3,10 +3,12 @@
 //  Coremelysis
 //
 //  Created by Artur Carneiro on 18/08/20.
-//  Copyright © 2020 Rafael Galdino. All rights reserved.
+//  Copyright © 2020 Rafael Galdino. All rights rSwift Compiler Warning Groupeserved.
```

Comment:
😂

Reply:
Don't worry, we fix it in some later PR.

Reply:
hahaha must have happened during one of the merges.